build.py

```diff
@@ -1555,6 +1555,11 @@ def core_build(cmake_script, repo_dir, cmake_dir, build_dir, install_dir,
         cmake_script.cp(
             os.path.join(repo_install_dir, 'lib', 'libtritonserver.so'),
             os.path.join(install_dir, 'lib'))
+        # Copy boost filesystem library used to compile tritonserver
+        cmake_script.mkdir(os.path.join(install_dir, 'boost_filesystem'))
+        cmake_script.cp(
+            os.path.join('/usr', 'lib', 'libboost_filesystem.so.1.80.0'),
+            os.path.join(install_dir, 'boost_filesystem'))
```

Comment:
Are we going to update our `x86` and `sbsa` images?

Reply:
This is specifically just for our Jetson Xavier platform, which we release as a tar file.

Reply:
What happen if someone will decide to rebuild `x86/sbsa` container from source, will it work as expected?

Reply:
As I understand it, Jetson Xavier is exclusively an `aarch64` [platform](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-xavier-series/#:~:text=View%20Jetson%20Xavier%20Technical%20Specifications). Any attempt to build an image using this branch outside the scope of this platform would be improper usage.

Are you asking me to put up guardrails in the build script to ensure this branch is not used for unsupported platforms? All this said, I don't believe any of these changes would be breaking for x86 if built from source.